### DISCUSSION TITLE
Google form option

### DISCUSSION THREAD

**Discussion by jenningsb2** [outline/outline#3128] [Ideas]
Hey team! 

I have a google form that I would love to use to capture feedback and interest on my documentation. I see that Google Docs/Sheets/Slides have an embedded option. 

**Would an embedded Google Form be an option?**

Since my company uses Google Workspace, it would be great to be able to stay in their ecosystem.

**Comment by tommoor** [comment]
It's not currently an option but that would be a solid addition, I've created an issue to track this: https://github.com/outline/outline/issues/3129

**Reply by jenningsb2** [reply]
Thanks Tom! Looking forward to it.
